model/host/hostutil.go

```diff
@@ -1067,11 +1067,11 @@ func (h *Host) AgentCommand(settings *evergreen.Settings) []string {
 	return []string{
 		h.Distro.AbsPathCygwinCompatible(h.Distro.HomeDir(), h.Distro.BinaryName()),
 		"agent",
-		fmt.Sprintf("--api_server='%s'", settings.ApiUrl),
-		fmt.Sprintf("--host_id='%s'", h.Id),
-		fmt.Sprintf("--host_secret='%s'", h.Secret),
-		fmt.Sprintf("--log_prefix='%s'", filepath.Join(h.Distro.WorkDir, "agent")),
-		fmt.Sprintf("--working_directory='%s'", h.Distro.WorkDir),
+		fmt.Sprintf("--api_server=%s", settings.ApiUrl),
```

Comment:
we want to remove the shell quotes for direct exec (as per your explanation in slack)

Reply:
Also this would break for arguments with spaces in them, such as windows paths. But fixing this is more important.